apps/video/index.html

```diff
@@ -51,12 +51,12 @@
     <div id="app-container">
       <div id="two-column-spearator"></div>
       <section role="region" id="thumbnail-views">
-        <gaia-header ignore-dir id="picker-header" action="back" class="thumbnails-list">
+        <gaia-header ignore-dir id="picker-header" action="back" class="thumbnails-list" no-font-fit>
```

Comment:
I noticed we are missing code to remove the attribute 'no-font-fit' to trigger gaia header font-fit logic after we hit 'visuallyloaded' mark, could that be the reason .